README.markdown

```diff
@@ -24,3 +24,51 @@ Quero ajudar!
 
 [![Bitdeli Badge](https://d2weczhvl823v0.cloudfront.net/rennerocha/dojopuzzles/trend.png)](https://bitdeli.com/free "Bitdeli Badge")
 
+Rodando o projeto!
+----------------------
+
+### Requeriments
+
+   - Python 3.7
+   - Django 3
+
+### Instalando a VirtualEnv
+
+```sh
+$ virtualenv --python=python3 venv
+$ source venv/bin/Activate
+```
+
+### Instalando o projeto
+
+```sh
+$ make install
+```
+
+
+### Rodando o serviço
+```sh
+$ make run
+```
+
+## Criador
+
+* **Renne Rocha** - *Initial work* - [rennerocha](https://github.com/rennerocha)
+
+### Apoiadores
```

Comment:
**suggestion**: We have more contributors that these two. I would prefer to keep the names out and just add a link to https://github.com/dojopuzzles/dojopuzzles/graphs/contributors (as you did in the end of the section).